packages/nouns-webapp/src/components/DynamicQuorumInfoModal/index.tsx

```diff
@@ -79,20 +79,20 @@ const DynamicQuorumInfoModalOverlay: React.FC<{
       <div className={classes.modal}>
         <div className={classes.content}>
           <h1 className={classes.title}>
-            <Trans>Dynamic Quorum</Trans>
+            <Trans>Dynamic Threshold</Trans>
           </h1>
 
           <p className={classes.mainCopy}>
             {window.innerWidth < 1200 ? (
               <Trans>
-                The Quorum (minimum number of For votes required to pass a proposal) is set as a
+                The Threshold (minimum number of For votes required to pass a proposal) is set as a
                 function of the number of Against votes a proposal has recieved. It increases
                 quadratically as a function of the % of Nouns voting against a prop, varying between
```

Comment:
unrelated to this change, but just noticed it says "It increases **quadratically** ..."
@bcjgit can you change it to *linearly* instead?